flask_security/utils.py

```diff
@@ -367,7 +367,11 @@ def get_token_status(token, serializer, max_age=None):
         user = _datastore.find_user(id=data[0])
 
     expired = expired and (user is not None)
-    return expired, invalid, user
+
+    if return_data:
+        return expired, invalid, user, data
```

Comment:
I hope this isn't too weird, but I did it in the name of backward compatibility and attempting to make the most minimal change possible.